Makefile

```diff
@@ -50,16 +51,19 @@ generate: packr2
 	$(GO) mod tidy
 	$(GO) generate ./...
 
-HAS_PACKR2 := $(shell command -v packr2)
-HAS_GOBIN_IN_PATH := $(shell re='(:|^)$(CLIENT_GOPATH)/bin/?(:|$$)'; if [[ "$${PATH}" =~ $${re} ]];then echo $${GOPATH}/bin;fi)
+HAS_PACKR2 := $(shell command -v packr3 && echo "OK")
 packr2:
 ifndef HAS_PACKR2
-ifndef HAS_GOBIN_IN_PATH
-	$(error "$(CLIENT_GOPATH)/bin is not in path and packr2 is not installed. Install packr2 or add "$(CLIENT_GOPATH)/bin to your path")
-endif
-	curl -SLo /tmp/packr.tar.gz https://github.com/gobuffalo/packr/releases/download/v2.6.0/packr_2.6.0_$(CLIENT_PLATFORM)_$(CLIENT_ARCH).tar.gz
-	cd /tmp && tar -xzf /tmp/packr.tar.gz
-	install /tmp/packr2 $(CLIENT_GOPATH)/bin/
+	mkdir -p tmp/packr2
```

Comment:
For my understanding, is `/tmp` not available for certain setups?

Are there any valid alternatives to creating the local `tmp` directory as done here?  Perhaps `TMPDIR=$(mktemp -d)` or some such?

Reply:
I can try that.

Reply:
I tried that. I think that having the tmp folder locally is better than creating a new temp folder on every build.